udata/core/user/api.py

```diff
@@ -316,6 +316,25 @@ def delete(self, user):
         return '', 204
 
 
+from udata.models import ContactPoint
+from udata.core.contact_point.api import ContactPointApiParser
+from udata.core.contact_point.api_fields import contact_point_page_fields
+
+
+contact_point_parser = ContactPointApiParser()
+
+
+@ns.route('/<user:user>/contact/', endpoint='user_contact_points')
```

Comment:
I'm a bit confused between `contact` vs `contacts`. Shouldn't it be plural for org and users?